src/xcode/ENA/ENA/Source/Views/BorderView.swift

```diff
@@ -0,0 +1,19 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import UIKit
+
+// Simple View which holds a property that indicates if this view was already drawn
+class BorderView: UIView {
```

Comment:
Hmm your implementation uses this as a BorderView but what really makes this view is the `wasDrawn` flag
```suggestion
class WasDrawnView: UIView {
```
But someone this does not sound nice, maybe you have an other Idea for naming?

Reply:
I know what you mean. What about `SingleUseView` or `StatefulView` @marcussc?